### PR TITLE
At/cb246/minors 7

### PR DESCRIPTION
**Pull request by Angelbt12**
https://inube.atlassian.net/jira/software/projects/CB/boards/20?selectedIssue=CB-246

### CODE REVIEW

File: src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx [hunk 1]
```diff
@@ -31,11 +31,11 @@ import {
...
 } from "@utils/formatData/text";
-import { formatISODatetoCustomFormat } from "@utils/formatData/date";
+import { formatPrimaryDate } from "@utils/formatData/date";
 import { currencyFormat } from "@utils/formatData/currency";
 import { ICreditProductProspect, Requests } from "@services/types";
 import { MenuPropect } from "@components/navigation/MenuPropect";
-import { menuOptions ,incomeOptions} from "./config/config";
+import { menuOptions, incomeOptions } from "./config/config";
 import { extraordinaryInstallmentMock } from "@mocks/prospect/extraordinaryInstallment.mock";
 import { ExtraordinaryPaymentModal } from "@src/pages/prospect/components/ExtraordinaryPaymentModal";
```
Comment (SelsaErojas) [inline src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx:40]: Adjust the import aliases using the alias for components and adjust order of the import.

File: src/pages/board/outlets/financialReporting/management/styles.ts [hunk 2]
```diff
@@ -10,9 +10,11 @@ export const ChatContent = styled.div`
...
   flex-direction: column;
+  padding-right: 10px; 
+  margin-right: -8px;
 
   &::-webkit-scrollbar {
-    border-radius: 8px;
+    width: 8px;
   }
 
   &::-webkit-scrollbar-thumb {
```
Comment (SelsaErojas) [inline src/pages/board/outlets/financialReporting/management/styles.ts:20]: Adjust to the inubekit library

**Review by SelsaErojas — CHANGES_REQUESTED** [review]
you should see this error in the requirements section
image

File: src/pages/board/outlets/financialReporting/management/styles.ts (hunk 2 above)
Comment (Angelbt12) [reply src/pages/board/outlets/financialReporting/management/styles.ts:20]: This file was already approved a long time ago, I just had to adjust the width of the scroll

Comment (SelsaErojas) [reply src/pages/board/outlets/financialReporting/management/styles.ts:20]: Do not use @inube/design-system

File: src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx [hunk 3]
```diff
@@ -31,7 +31,7 @@ import {
   capitalizeFirstLetter,
   capitalizeFirstLetterEachWord,
 } from "@utils/formatData/text";
-import { formatISODatetoCustomFormat } from "@utils/formatData/date";
+import { formatPrimaryDate } from "@utils/formatData/date";
```
Comment (SelsaErojas) [inline src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx:34]: Adjust the import aliases using the alias for components and adjust order of the import. (line 38 and line 40)

**Review by SelsaErojas — CHANGES_REQUESTED** [review]

**Review by SelsaErojas — APPROVED** [review]
Funtional

**Review by carin91 — APPROVED** [review]

**Pull request merged**